scripts/qa_crowbarsetup.sh

```diff
@@ -4659,9 +4744,9 @@ function onneutron_wait_for_neutron
     [ -z "$NEUTRON_SERVER" ] && return
 
     wait_for 300 3 "ssh $NEUTRON_SERVER 'rcopenstack-neutron status' |grep -q running" "neutron-server service running state"
-    wait_for 200 3 " ! ssh $NEUTRON_SERVER '. .openrc && neutron --insecure agent-list -f csv --quote none'|tail -n+2 | grep -q -v ':-)'" "neutron agents up"
+    wait_for 200 3 " ! ssh $NEUTRON_SERVER '. .openrc && neutron agent-list -f csv --quote none'|tail -n+2 | grep -q -v ':-)'" "neutron agents up"
```

Comment:
can we change this to use the openstack command? then it would work either way also for cloud6 and older

Reply:
I'm not sure if the openstack CLI has the agent-list command available in cloud6, and in any case we would still need to conditionally use --insecure

Reply:
yeah, looks like that wasn't in liberty. too bad.